config/notifiers.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/pkg/errors"
 
 	commoncfg "github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
```

Comment:
Please don't add any dependencies on model.

Reply:
I removed the dependency.